R/expand.R

```diff
@@ -56,6 +56,29 @@ locf <- function(x) {
 #' `expand_locf()` a simple composition function of `fill_locf()` and `expand()` i.e.
 #' `fill_locf(expand(...))`.
 #'
+#' ## Missing First Values
+#'
+#' The `fill_locf()` function performs last observation carried forward imputation.
+#' A natural consquence of this is that it is unable to impute missing observations if the
+#' observation is the first value.
+#' A common request is for this function to impute the first value if it is missing
+#' however doing so risks silent errors in the case of time varying covariates.
```

Comment:
```suggestion
#' A common request for this function is to impute the value of a baseline covariate if the row corresponding to the first observation for a subject is missing.
#' However doing so risks silent errors in the case of time varying covariates, so it is not allowed.
#' Perhaps the easiest in practice is to use a LOCF imputation with the baseline visit included (which should never be a missing row) and then remove the baseline afterwards (in case the change from baseline is modeled).
```

Reply:
@nociale , Sorry I'm not convinced by this wording. Was there a particular message you were trying to convey ?  In general in documentation I feel like we should try to avoid words like "perhaps", "maybe" , "potentially" and instead make it seem more authoritative.

Reply:
I agree "perhaps" should be avoided